pro-homeworks/hw02-collections/src/main/java/homework/CustomerService.java

```diff
@@ -0,0 +1,28 @@
+package homework;
+
+import java.util.AbstractMap;
+import java.util.Map;
+import java.util.TreeMap;
+
+
+public class CustomerService {
+    private final TreeMap<Customer, String> customers = new TreeMap<>();
```

Comment:
```suggestion
    private final NavigableMap<Customer, String> customers = new TreeMap<>();
```

Тип лучше объявлять минимально необходимым интерфейсом.